distribution/docs/src/main/resources/_contents/_solr-contents/managing-solr-contents.adoc

```diff
@@ -309,3 +309,50 @@ Here are some helpful suggestions for setting up data backups and recoveries:
 * Run a scheduler program that calls the backup URL on a timed basis.
 * Put indicators in place that can detect when data corruption may have occurred.
 * Testing a backup before recovery is possible. A replicated "staging" Solr server instance can be stood up, and the backup can be copied to that system for testing before moving it to the "production" system.
+
+=== High Availability (Optional)
+
+==== SolrCloud
+
+SolrCloud provides distributed capabilities of a highly available, fault tolerant cluster of Solr servers.
+
```

Comment:
I'm not sure if this should be `SolrCloud provides the distributed capabilities of a highly available, fault tolerant cluster of Solr servers.`
or
`SolrCloud provides distributed capabilities through a highly available, fault tolerant cluster of Solr servers.`

It depends on the intended meaning of the sentence.